### PR TITLE
Create documentation on how to create a hotfix

### PR DESCRIPTION
**Pull request by jeffaco**
@MSFTOSSMgmt/ostc-devs

### CODE REVIEW

**Comment by ericgable** [comment]
Wow!  Looks great to me!  Sign Off.

File: workflow-hotfix.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
+# Workflow for Creating a Hotfix Release
+
+Creating a hotfix for a release is very simple. There are basic steps
```
Comment (niroyb) [inline workflow-hotfix.md]: Remove redundant "There are basic steps to follow to create a hotfix release"

File: workflow-hotfix.md [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+* 2c3cb16 Update readme.md
+* 6a41725 Update readme.md
+* 087b08f Correct Debian version format of OMI to x.y.z.r (rather than x.y.z-r)
+* d0e6421 (tag: v1.1.1-70) Fix up log rotation for OMI (consider moving to OMI project later)
+* b88c2f7 Explicitly disable -Werror in ./configure
+* 3b8679c Fix compile warnings in EngineHelper
+* 499a8a5 Add instructions to build from source
+>```
+
+Say we want to not ship our hotfix the first three changes, instead
```
Comment (niroyb) [inline workflow-hotfix.md]: Rephrase to "Let's say we don't want to ship the three most recent commints in the hotfix"

**Comment by niroyb** [comment]
Signed off :+1:

**Pull request merged**